### ISSUE TITLE
Feature Request: Support for RTP streams

### ISSUE THREAD

**Issue by chris6570** [SenexCrenshaw/StreamMaster#18]
**Is your feature request related to a problem? Please describe.**
StreamMaster cannot currently accept RTP streams as input sources. Attempting to use them results in the following log error: "Error getting proxy stream for rtp://87.141.215.251@232.0.10.221:10000: The 'rtp' scheme is not supported." 

**Describe the solution you'd like**
Support for RTP streams when used as source URLs.

**Describe alternatives you've considered**
None.

**Additional context**
Please let me know if log files or other information is needed or would be helpful.

**Comment by chris6570** [comment]
Yes. If you need others or need me to test anything, please let me know.

**Comment by SenexCrenshaw** [comment]
hmm. i cant seem to get this to work in vlc.
You just use:
``rtp://87.141.215.251@232.0.10.221:10000`` ?

**Comment by chris6570** [comment]
Yes. However, it is stream from a German broadcaster which might not be available to you. In fact, all of the RTP streams that I use are from German providers. I should have highlighted that to you. Sorry for not doing so.

**Comment by SenexCrenshaw** [comment]
no worries. I need to find a rtp stream to test with.

**Comment by SenexCrenshaw** [comment]
you could try using ffmpeg as the proxy. I havent gotten a chance to find a rtsp stream yet

**Comment by chris6570** [comment]
I will try to do this weekend and report back to you. Thank you for continuing to pursue this.
